frontend/src/metabase/dashboard/containers/DashboardHeaderContainer.jsx

```diff
@@ -55,7 +55,7 @@ const mapDispatchToProps = {
   addActionToDashboard,
 };
 
-class DashboardHeader extends Component {
+class DashboardHeaderContainer extends Component {
```

Comment:
@EmmadUsmani the existing default way currently is to name container components the same as unconnected components which surely can be confusing. I think we should not introduce the new way in a single place but enforce it as a new rule if we want to switch

Reply:
@alxnddr Following our [discussion on slack](https://metaboat.slack.com/archives/C505ZNNH4/p1680222581340519?thread_ts=1680196503.621509&cid=C505ZNNH4) I've updated the PR to use the agreed upon convention of `Component` and `ComponentView`